wgpu/src/lib.rs

```diff
@@ -2129,6 +2130,7 @@ impl Device {
             context: Arc::clone(&self.context),
             id: Context::device_create_buffer(&*self.context, &self.id, desc),
             map_context: Mutex::new(map_context),
+            size: desc.size,
```

Comment:
Long term we should think of we should be storing this stuff at the surface, but for now this is fine.